publish-frontend/src/pages/index.js

```diff
@@ -1,52 +1,231 @@
+                  <MenuItemOption value='all'>All posts</MenuItemOption>
+                  <MenuItemOption value='drafts'>Drafts posts</MenuItemOption>
+                  <MenuItemOption value='published'>
+                    Published posts
+                  </MenuItemOption>
+                </MenuOptionGroup>
+              </MenuList>
+            </Menu>
+            <Menu>
+              <FilterButton text='All authors' />
+              <MenuList>
+                <MenuOptionGroup defaultValue='all' type='radio'>
+                  <MenuItemOption value='all'>All authors</MenuItemOption>
+                  {users.map(user => (
+                    <MenuItemOption key={user.id} value={user.id}>
+                      {user.name}
+                    </MenuItemOption>
+                  ))}
+                </MenuOptionGroup>
+              </MenuList>
+            </Menu>
+            <Menu>
+              <FilterButton text='All tags' />
+              <MenuList>
+                <MenuOptionGroup defaultValue='all' type='radio'>
+                  <MenuItemOption value='all'>All tags</MenuItemOption>
+                  {tags.data.map(tag => (
+                    <MenuItemOption key={tag.id} value={tag.id}>
+                      {tag.attributes.name}
+                    </MenuItemOption>
+                  ))}
+                </MenuOptionGroup>
+              </MenuList>
+            </Menu>
+            <Menu>
+              <FilterButton text='Sort by: Newest' />
+              <MenuList>
+                <MenuOptionGroup defaultValue='newest' type='radio'>
+                  <MenuItemOption value='newest'>Newsest</MenuItemOption>
+                  <MenuItemOption value='oldest'>Oldest</MenuItemOption>
+                  <MenuItemOption value='recently-updated'>
+                    Recently updated
+                  </MenuItemOption>
+                </MenuOptionGroup>
+              </MenuList>
+            </Menu>
+          </Grid>
+
+          <Box pb='10'>
+            <Table boxShadow='md' borderWidth='1px'>
+              <Thead bgColor='rgb(243, 244, 246)'>
+                <Tr>
+                  <Th>Title</Th>
+                  <Th w='140px' display={{ base: 'none', sm: 'table-cell' }}>
+                    Status
+                  </Th>
+                </Tr>
+              </Thead>
+              <Tbody bgColor='white'>
+                {posts.data.map(post => {
+                  const title = post.attributes.title;
+                  const name = post.attributes.author.data.attributes.name;
+                  const tag = post.attributes.tags.data[0]?.attributes.name;
+                  console.log(post.attributes.tags);
+                  const relativeUpdatedAt = intlFormatDistance(
+                    new Date(post.attributes.updatedAt),
+                    new Date()
+                  );
+                  const status =
+                    post.attributes.publishedAt !== null ? (
```

Comment:
```suggestion
                    post.attributes.publishedAt ? (
```
Is `publishedAt` a date string? If so, this should be equivalent.